schemas/com.getvero/bounced/jsonschema/1-0-0

```diff
@@ -12,25 +12,60 @@
 	"properties": {
 		"bounced_at": {
 			"type": "string",				
-			"format": "date-time"
+			"format": "date-time",
+			"maxLength": 10
 		},
 		"bounce_type": {
-			"type": "string",
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"bounce_code": {
+			"type": ["string", "null"],
+			"maxLength": 32
+		},
+		"bounce_message": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"message_id": {
+			"type": ["string", "null"],
 			"maxLength": 255
 		},
 		"type": {
 			"type": "string",
 			"maxLength": 12
```

Comment:
Please extend the `maxLength` for all `type` fields just for safety, since e.g. the "user_unsubscribed" type might be renamed to something like "user_successfully_unsubscribed" in the future which would break the schema